caasp-389-ds-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -27,38 +27,69 @@ if [ ! -e "/data/config/setup.inf" ]; then
 
     echo -e "\n>> Creating localhost configuration...\n"
     cat <<EOT >> /data/config/setup.inf
-[General]
-FullMachineName=localhost.${DOMAIN_SUFFIX}
-;SuiteSpotUserID=dirsrv
-;SuiteSpotGroup=dirsrv
-;Should not run as root
-SuiteSpotUserID=root
-;Should not run as root
-SuiteSpotGroup=root
-StrictHostCheck=false
+[general]
+config_version = 2
+;defaults = 999999999
+;full_machine_name = localhost.${DOMAIN_SUFFIX}
```

Comment:
The previous setting is set but the new is commented out. Is this on purpose? because the value will be set by default?

Reply:
From the template description, we do not need it since `strict_host_checking` is set to `false`.
```
# full_machine_name (str)
# Description: Sets the fully qualified hostname (FQDN) of this system. When installing this instance with GSSAPI authentication behind a load balancer, set this parameter to the FQDN of the load balancer and, additionally, set "strict_host_checking" to "false".
# Default value: 389-ds
;full_machine_name = 389-ds
```